### PR TITLE
Bugfix -- filterControl -- trim values for select fields

### PR DESCRIPTION
**Pull request by ioctaptceb**
fixes issue: #1728 -- full description can be found there

solution: just trim the value before setting it.

added a couple of formatting changes according to the linter as well

### CODE REVIEW

**Comment by djhvscf** [comment]
Great! Thanks @ioctaptceb

**Comment by djhvscf** [comment]
Fix #1728

File: src/extensions/filter-control/bootstrap-table-filter-control.js [hunk 1]
```diff
@@ -11,6 +11,7 @@
     var sprintf = $.fn.bootstrapTable.utils.sprintf;
 
     var addOptionToSelectControl = function (selectControl, value, text) {
+        value = value.trim();
```
Comment (wenzhixin) [inline src/extensions/filter-control/bootstrap-table-filter-control.js]: Please use `$.trim(value)` instead because IE8 does not support this function.

**Comment by wenzhixin** [comment]
Thanks

**Pull request merged**